ncl/ncl_entries/day_of_week.ipynb

```diff
@@ -0,0 +1,232 @@
+    "import cftime\n",
+    "\n",
+    "day = 4\n",
+    "month = 6\n",
+    "year = 2024\n",
+    "\n",
+    "dow = cftime.datetime(\n",
+    "    year, month, day, calendar='proleptic_gregorian', has_year_zero=True\n",
+    ").strftime('%A')\n",
+    "print(f\"Proleptic Gregorian calendar: \\t\\t{year}-{month}-{day} is a {dow}\")\n",
+    "\n",
+    "dow = cftime.datetime(\n",
+    "    year, month, day, calendar='standard', has_year_zero=True\n",
+    ").strftime('%A')\n",
+    "print(f\"Standard calendar: \\t\\t\\t{year}-{month}-{day} is a {dow}\")\n",
+    "\n",
+    "dow = cftime.datetime(year, month, day, calendar='julian', has_year_zero=True).strftime(\n",
+    "    '%A'\n",
+    ")\n",
+    "print(f\"Julian calendar: \\t\\t\\t{year}-{month}-{day} is a {dow}\")\n",
+    "\n",
+    "dow = cftime.datetime(\n",
+    "    year, month, day, calendar='360_day', has_year_zero=True\n",
+    ").strftime('%A')\n",
+    "print(f\"360 day calendar: \\t\\t\\t{year}-{month}-{day} is a {dow}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Differences between `cftime` and NCL's `day_of_week`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Calendars\n",
+    "NCL's `day_of_week` only supports the proleptic Gregorian calendar, while the `cftime` module supports all CF conventions calendars, including the `proleptic_gregorian` calendar."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Input Type\n",
+    "Notably, using `cftime` to calculate the day of the week works by getting a `dayofwk` value from a `cftime.datetime` object, which means that the calculations for the day of the week have to be collected through each date individually, while NCL's `days_in_month` can take in multidimensional integer arrays, given that the `year` and `month` arrays have the same dimensions."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Return Value\n",
+    "There is a slight difference in the resulting value between NCL's `day_of_week` and `cftime`/`datetime`. \n",
+    "- NCL's `day_of_week` returns an integer array of the same size as `year` with values between 0 and 6 where, 0=Sunday, 1=Monday, ...\n",
+    "- By default, `cftime` and `datetime` return an integer between 0 and 6, where, 0=Monday, 1=Tuesday, ... \n",
+    "    - This can be corrected for by changing the `strftime` representation, which adjusts the result to agree with NCL's `day_of_week`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Year 0\n",
+    "There is a slight difference in the way that NCL's `day_of_week` and the `cftime` module handle the year 0.\n",
+    "- NCL's `day_of_week` supports all positive years by default, including year 0.\n",
+    "- The `cftime` module also supports all positive years including year 0, as long as you specify `has_year_zero=True`."
```

Comment:
```suggestion
    "- The `cftime` module also supports all positive years, but handling for year 0 is dependent upon the calendar and/or the `has_year_zero` keyword argument."
```

There's a little bit of nuance here w/ the various calendars and how that `has_year_zero` kwarg works with them.  I think this should be good enough though.